client-java/sql/src/main/java/org/evomaster/client/java/sql/SchemaExtractor.java

```diff
@@ -557,11 +596,11 @@ private static void handleTableEntry(Connection connection, DbSchemaDto schemaDt
 
         tableDto.primaryKeySequence.addAll(primaryKeySequence.values());
 
+        //ResultSet columns = md.getColumns(null, schemaDto.name, tableDto.name, null);
```

Comment:
isn't this commented line identically to the following one? shouldn't it be removed?

Reply:
yes. Copy paste error.